README.md

```diff
@@ -9,3 +9,9 @@ YOUR NAME HERE:
 1. A truth (or a lie)
 2. A lie (or a truth)
 3. Another truth (or your lie)
+
+
+Isabelle
+1. I have been trampled by a horse
+2. I was born with red hair
```

Comment:
I don't think you were born with red hair. It was uhh.. blonde or something

Reply:
i concur but am horrified about the ocean